packages/framework/src/Component/ClassExtension/MethodAnnotationsFactory.php

```diff
@@ -63,16 +62,15 @@ public function getMethodAnnotationLine(
         ReflectionClass $projectClassBetterReflection
     ): string {
         foreach ($this->annotationsReplacementsMap->getPatterns() as $frameworkClassPattern) {
-            foreach ($reflectionMethodFromFrameworkClass->getDocBlockReturnTypes() as $docBlockReturnType) {
-                if (!$this->isMethodImplementedInClass($reflectionMethodFromFrameworkClass->getName(), $projectClassBetterReflection)
-                    && $this->isMethodReturningTypeThatIsExtendedInProject($frameworkClassPattern, $docBlockReturnType)
-                ) {
+            if (!$this->isMethodImplementedInClass($reflectionMethodFromFrameworkClass->getName(), $projectClassBetterReflection)) {
+                if ($this->isMethodReturningTypeThatIsExtendedInProject($frameworkClassPattern, $reflectionMethodFromFrameworkClass->getDocBlockReturnTypes())
+                    || $this->methodParameterTypeIsExtendedInProject($frameworkClassPattern, $reflectionMethodFromFrameworkClass->getParameters())) {
```

Comment:
can you unify the names of methods? 
From 
`isMethodReturningTypeThatIsExtendedInProject` - `methodParameterTypeIsExtendedInProject`
to 
`methodReturningTypeIsExtendedInProject` - `methodParameterTypeIsExtendedInProject`

Reply:
done in 3e181880b